03Lesson/01Project/css/style.css

```diff
@@ -1 +1,51 @@
 
+body {
+  font-family: 'Arial', sans-serif;
+}
+
+header, main, footer {
+  width: 1024px;
+  font-family: sans-serif;
+  margin: auto;
+}
+
+header {
+  background-color: #0071c5;
+  padding: 10px;
+  color: white
+}
+
+header-#h1 {
+  font-weight: 100;
+  font-size: 60px;
+  color: #fff;
+}
+
+.nav-link {
+  color: white;
+  font-size: 25px;
+  margin-right: 15px;
+}
+
+main aside h3 {
+  font-size: 24px;
+  font-weight: 200;
+}
+
+main > p {
+  font-size: 16px;
+  color: #555;
+}
+
+body html aside {

```

Comment:
This selector does not make sense. There can never be a `html` element inside of a `body` element in valid HTML markup.